tox.ini

```diff
@@ -15,6 +16,7 @@ deps =
     dj4: Django>=4,<5
     wagtail215: wagtail>=2.15,<2.16
     wagtail216: wagtail>=2.16,<2.17
+    wagtail30: wagtail>=3.0rc1,<4
```

Comment:
**Suggestion:** Is Wagtail changing its versioning scheme or should we do `<3.1` instead?

Reply:
Wagtail switched to abiding by SemVer more closely.
The next release will be 4.0 due to the second batch of admin interface changes. They cover lots of areas, so in keeping up with SemVer, it will be a 4.0

Reply:
I have changed this to `wagtail>=3.0,<4.0`, the next release will be 4.0 based on the wagtail doc site.

Reply:
So instead `wagtail30`, should this just be `wagtail3: wagtail>=3.0rc1,<4`?

Reply:
that actually makes sense, it should be wagtail3 instead, also wagtail 3.0 is out, we don't need `rc1` anymore.